src/test/scala/uk/gov/nationalarchives/tdr/api/service/FileServiceSpec.scala

```diff
@@ -21,58 +20,67 @@ class FileServiceSpec extends AnyFlatSpec with MockitoSugar with Matchers {
+    val expectedArgs = List(fileRowOne, fileRowTwo, fileRowThree)
     val captor: ArgumentCaptor[List[FileRow]] = ArgumentCaptor.forClass(classOf[List[FileRow]])
     val mockResponse = Future.successful(List(fileRowOne, fileRowTwo, fileRowThree))
 
     when(fileRepositoryMock.addFiles(captor.capture())).thenReturn(mockResponse)
-    val fileService = new FileService(fileRepositoryMock, FixedTimeSource)
-    val result: Files = fileService.addFile(AddFilesInput(consignmentId, 3),Some(uuid)).await()
+    val fileService = new FileService(fileRepositoryMock, FixedTimeSource, fixedUuidSource)
+    val result: Files = fileService.addFile(AddFilesInput(consignmentUuid, 3),Some(userUuid)).await()
 
     captor.getAllValues.size should equal(1)
     captor.getAllValues.get(0).length should equal(3)
-    captor.getAllValues.get(0) should equal(expectedArgs)
+    captor.getAllValues.get(0).forall(expectedArgs.contains) should be(true)
 
-    result.fileIds shouldBe List(1,2,3)
+    result.fileIds shouldBe List(fileUuidOne, fileUuidTwo, fileUuidThree)
   }
 
   "createFile" should "link a file to the correct user and consignment" in {
+    val fixedUuidSource = new FixedUUIDSource()
     val userId = UUID.randomUUID()
-    val consignmentId = 123
+    val fileUuid = fixedUuidSource.uuid
+    val consignmentUuid = UUID.randomUUID()
     val fileRepositoryMock = mock[FileRepository]
-    val fileService = new FileService(fileRepositoryMock, FixedTimeSource)
+    fixedUuidSource.reset
+    val fileService = new FileService(fileRepositoryMock, FixedTimeSource, fixedUuidSource)
 
-    val expectedRow = List(FileRow(consignmentId, userId.toString, Timestamp.from(FixedTimeSource.now), None))
-    val expectedArgs = List(FileRow(consignmentId, userId.toString, Timestamp.from(FixedTimeSource.now), Option.empty))
+    val expectedRow = List(FileRow(fileUuid, consignmentUuid, userId, Timestamp.from(FixedTimeSource.now)))
     val captor: ArgumentCaptor[List[FileRow]] = ArgumentCaptor.forClass(classOf[List[FileRow]])
-    val mockResponse = Future.successful(List(FileRow(consignmentId, userId.toString, Timestamp.from(FixedTimeSource.now), Some(1))))
+    val mockResponse = Future.successful(List(FileRow(fileUuid, consignmentUuid, userId, Timestamp.from(FixedTimeSource.now))))
     when(fileRepositoryMock.addFiles(captor.capture())).thenReturn(mockResponse)
 
-    fileService.addFile(AddFilesInput(consignmentId, 1),Some(userId)).await()
+    fileService.addFile(AddFilesInput(consignmentUuid, 1),Some(userId)).await()
 
     verify(fileRepositoryMock).addFiles(expectedRow)
     captor.getAllValues.size should equal(1)
-    captor.getAllValues.get(0).head.consignmentid should equal(consignmentId)
-    captor.getAllValues.get(0).head.userid should equal(userId.toString)
+    captor.getAllValues.get(0).head.consignmentid should equal(consignmentUuid)
+    captor.getAllValues.get(0).head.userid should equal(userId)
```

Comment:
Unnecessary line break

Reply:
I've removed this one and the one below.